.markdownlint-cli2.jsonc

```diff
@@ -0,0 +1,14 @@
+// https://github.com/DavidAnson/markdownlint-cli2#markdownlint-cli2jsonc
+{
+  "config": {
+    "MD013": false,
+    "MD024": false,
+    "MD033": false,
+    "MD041": false,
+    "MD026": false,
```

Comment:
This one as well looks quite nice to apply and fix in our markdowns.

[MD026](https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md026.md) no-trailing-punctuation - Trailing punctuation in heading

Reply:
Again, I'm calling out of scope for this PR.

The current failures are:

```
.github/ISSUE_TEMPLATE/feature_request.md:9:66 MD026/no-trailing-punctuation Trailing punctuation in heading [Punctuation: '.']
.github/PULL_REQUEST_TEMPLATE.md:39:13 MD026/no-trailing-punctuation Trailing punctuation in heading [Punctuation: ':']
```

That first one is this heading:

```
### Is your feature request related to a problem? Please describe.
```

Given it has two sentences in the heading, I think it is *good* to end with punctuation.